jerry-core/mem/mem-heap.cpp

```diff
@@ -40,12 +40,14 @@
 #ifdef JERRY_VALGRIND
 # include "memcheck.h"
 
-# define VALGRIND_NOACCESS_STRUCT(s)    (void)VALGRIND_MAKE_MEM_NOACCESS((s), sizeof (*(s)))
-# define VALGRIND_UNDEFINED_STRUCT(s)   (void)VALGRIND_MAKE_MEM_UNDEFINED((s), sizeof (*(s)))
-# define VALGRIND_DEFINED_STRUCT(s)     (void)VALGRIND_MAKE_MEM_DEFINED((s), sizeof (*(s)))
-# define VALGRIND_NOACCESS_SPACE(p, s)  (void)VALGRIND_MAKE_MEM_NOACCESS((p), (s))
-# define VALGRIND_UNDEFINED_SPACE(p, s) (void)VALGRIND_MAKE_MEM_UNDEFINED((p), (s))
-# define VALGRIND_DEFINED_SPACE(p, s)   (void)VALGRIND_MAKE_MEM_DEFINED((p), (s))
+# define VALGRIND_NOACCESS_STRUCT(s)     (void)VALGRIND_MAKE_MEM_NOACCESS((s), sizeof (*(s)))
+# define VALGRIND_UNDEFINED_STRUCT(s)    (void)VALGRIND_MAKE_MEM_UNDEFINED((s), sizeof (*(s)))
```

Comment:
Why do we need the `(void)` parts? With the parts, the code doesn't compile in my configuration.

Command line I've used: `make VERBOSE=1 VALGRIND=ON debug.linux -j`